packages/medusa/src/services/customer-group.ts

```diff
@@ -86,6 +89,38 @@ class CustomerGroupService extends BaseService {
     })
   }
 
+  /**
+   * Update a customer group.
+   *
+   * @param {string} customerGroupId - id of the customer group
+   * @param {CustomerGroupUpdate} update - customer group partial data
+   */
+  async update(
+    customerGroupId: string,
+    update: CustomerGroupUpdate
+  ): Promise<CustomerGroup[]> {
+    return this.atomicPhase_(async (manager) => {
+      const { metadata, ...properties } = update
+
+      const cgRepo: CustomerGroupRepository = manager.getCustomRepository(
+        this.customerGroupRepository_
+      )
+
+      const customerGroup = await this.retrieve(customerGroupId)
+
+      for (const key in properties) {
+        if (typeof properties[key] !== "undefined") {
+          customerGroup[key] = properties[key]
+        }
+      }
+
+      if ("metadata" in update) {
+        customerGroup.metadata = this.setMetadata_(customerGroup, metadata)
+      }
```

Comment:
Nitpicking: We are destructuring the `update` object on line 103 and pulling out the `metadata` property from `update` to use directly. If not present, `metadata` will be `undefined`. Taking these two observations into account, I think it would make sense to change the check to be:
```suggestion
      if (typeof metadata !== `undefined`) {
        customerGroup.metadata = this.setMetadata_(customerGroup, metadata)
      }
```

Reply:
Done :)